source/game/field/ObjectDirector.cc

```diff
@@ -8,6 +8,7 @@
 
 #include "game/system/CourseMap.hh"
 #include "game/system/RaceConfig.hh"
+#include <game/field/obj/ObjectShip64.hh>
```

Comment:
Notice that no other objects are included here. You should instead add this include to `ObjectRegistry.hh` and use quotes instead of angled brackets.